site/content/en/docs/Installation/Install Agones/helm.md

```diff
@@ -280,6 +281,67 @@ The following tables lists the configurable parameters of the Agones chart and t
+| `agones.allocator.service.http.enabled`                  | If true the [allocator service][allocator] will respond to [REST requests][rest-requests]                                                                                                                                                                                                                        | `true`                                    |
+| `agones.allocator.service.http.appProtocol`              | The `appProtocol` to set on the Service for the http allocation port. If left blank, no value is set.                                                                                                                                                                                                            | ``                                        |
+| `agones.allocator.service.http.port`                     | The port that is exposed externally by the [allocator service][allocator] for [REST requests][rest-requests]                                                                                                                                                                                                     | `443`                                     |
+| `agones.allocator.service.http.portName`                 | The name of exposed port                                                                                                                                                                                                                                                                                         | `http`                                    |
+| `agones.allocator.service.http.targetPort`               | The port that is used by the allocator pod to listen for [REST requests][rest-requests]. Note that the allocator server cannot bind to low numbered ports.                                                                                                                                                       | `8443`                                    |
+| `agones.allocator.service.http.nodePort`                 | If the ServiceType is set to "NodePort",  this is the NodePort that the allocator http service is exposed on.                                                                                                                                                                                                    | `30000-32767`                             |
+| `agones.allocator.service.http.unallocatedStatusCode`                 | HTTP status code to return when no GameServer is available for allocation.This setting allows for custom responses when a game server allocation fails, offering flexibility in handling these situations.                                                                                                                                                                                                    | `429`                             |
```

Comment:
Here's the change ⬆️. Please let me know if there's anything else that needs to be addressed.

Reply:
Nit: add a space to "allocation.This"

Reply:
I've made the correction.